### PR TITLE
Input field suffix

### PR DESCRIPTION
**Pull request by PavelHolec**
Add interactive suffix action to InputField

image

image

### CODE REVIEW

File: Sources/Orbit/Components/InputField.swift [hunk 1]
```diff
@@ -14,11 +14,12 @@ public struct InputField: View {
 
     @Binding private var value: String
     @Binding private var messageHeight: CGFloat
-    @State private var isPressed: Bool = false
     @State private var isEditing: Bool = false
 
     let label: String
     let placeholder: String
+    let prefix: Icon.Content
+    let suffix: Icon.Content
```
Comment (sjavora) [inline Sources/Orbit/Components/InputField.swift:22]: What is the logic in naming this one suffix, but calling it disclosure in other components? If it's "because Orbit calls it that", then I suggest making them unify it, because, this is just confusing.

File: Sources/Orbit/Support/Layout/Strut.swift [hunk 2]
```diff
@@ -5,7 +5,7 @@ import SwiftUI
 /// Used to fix layout issues where `frame(minHeight:)` causes issues.
 struct Strut: View {
     
-    let height: CGFloat
+    let height: CGFloat?
```
Comment (sjavora) [inline Sources/Orbit/Support/Layout/Strut.swift:8]: What is this change related to?

**Comment by sjavora** [comment]
See, this is a reason you don't name elements "suffix" 😄 

Screenshot 2022-02-04 at 10 38 38

File: Sources/Orbit/Support/Layout/Strut.swift (hunk 2 above)
Comment (sjavora) [reply Sources/Orbit/Support/Layout/Strut.swift:8]: Oh I see... the module currently doesn't build. I wonder how we got into this situation given that each pull request builds the package...

File: Sources/Orbit/Components/InputField.swift (hunk 1 above)
Comment (PavelHolec) [reply Sources/Orbit/Components/InputField.swift:22]: Yep, so far it is the inconsistency in Orbit docs :/ I would address it later. For now, it seemed better if it matches the relevant component naming (although the disclosure is yet another term which should be renamed) 

image

image

File: Sources/Orbit/Support/Layout/Strut.swift (hunk 2 above)
Comment (PavelHolec) [reply Sources/Orbit/Support/Layout/Strut.swift:8]: 🤷

**Comment by PavelHolec** [comment]
> See, this is a reason you don't name elements "suffix" 😄
> 
> Screenshot 2022-02-04 at 10 38 38

We need to tell Orbit then :D

**Comment by sjavora** [comment]
@PavelHolec well all my comments have to do with naming... AFAIK you are the go-to person when it comes to Orbit <-> Mobile/iOS communication, so please bring the inconsistencies up 🙏

**Review by sjavora — APPROVED** [review]
